sktime/transformations/series/summarize.py

```diff
@@ -219,6 +219,10 @@ class WindowSummarizer(BaseTransformer):
         # CI and test flags
         # -----------------
         "tests:core": True,  # should tests be triggered by framework changes?
+        # move skip config from tests._config to estimator tag per #8515
```

Comment:
please remove the comment that it has been moved, instead add a reference to the original issue that describes why it was skipped